### PR TITLE
Create User Research page “choosing-and-using-research-methods”

### PR DESCRIPTION
**Pull request by johnwaterworth**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by johnwaterworth** [comment]
Adding a new guide on Choosing and using research methods. This is a first version. We plan to cover more methods. And add links to tools and examples.
We reviewed this at the user researcher away day.

**Review by snim2 — APPROVED** [review]

File: src/user-research/choosing-and-using-research-methods.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+Choose interviews when:
+- your research is broad and you don’t yet know which topics will be most relevant or which questions will be most useful
+- the subject is complicated, will require some explanation and participants are likely to give you long and detailed answers
+- you want the chance to ask follow up questions to fully explore and better understand what participants tell you
+
+You can combine interviews with usability testing, by talking to participants to understand previous experiences and current use, before asking them to try out a prototype or service.
+
+### Recommended guidance
+- [Using in-depth interviews in the GOV.UK Service Manual](https://www.gov.uk/service-manual/user-research/using-in-depth-interviews)
+- [Interviewing Users by Steve Portigal] (https://portigal.com/Books/interviewing-users/)
```
Comment (yndajas) [inline src/user-research/choosing-and-using-research-methods.md:40]: This has a space between tghe  that breaks the brackets that breaks the lin

**Review by yndajas — COMMENTED** [review]
Just had a very quick scan and spotted a few things

File: src/user-research/choosing-and-using-research-methods.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+permalink: /:path/:basename/
+related_order: 70
+---
+Our fourth research principle is to [Be methodical, but not rigid](https://playbook.dxw.com/user-research/#user-research-principles). At dxw, we don’t have a fixed set of approved research methods. But when we [plan our research](https://playbook.dxw.com/user-research/creating-and-using-research-plans/), we do need to choose appropriate methods for the context and apply them well.
+
+We prefer tried and trusted methods that can provide strong evidence and reliable answers to our questions, for the least time, effort and cost.
+
+And we always consider a team’s level of experience with user research. So we may prefer simpler methods like interviews and usability testing that the team can easily understand and join in with.
+
+This guide lists our favourite books, articles and videos for different research methods. Provides links to any tools and templates we have created. Along with good examples from previous projects.
```
[the remark this answers is not in the document]
Comment (johnwaterworth) [reply src/user-research/choosing-and-using-research-methods.md]: I had clumsily broken up a previous sentence. This should be better.

**Pull request merged**